running-a-nats-service/configuration/mqtt/README.md

```diff
@@ -64,7 +64,7 @@ If the MQTT subscription is QoS1 and an MQTT publisher publishes an MQTT QoS1 me
 When the server delivers a QoS 1 message to a QoS 1 subscription, it will keep the message until it receives the PUBACK for the corresponding packet identifier. If it does not receive it within the "ack_wait" interval, that message will be resent.
 
 ## Max Ack Pending
-
+z
```

Comment:
This should be removed and not updated unless you had some other correction in mind.

Reply:
fixed:)